hack/log/log-dump.sh

```diff
@@ -191,3 +200,5 @@ else
     echo "================ DUMPING LOGS FOR WORKLOAD CLUSTER (Windows) ========"
     dump_workload_cluster_logs_windows
 fi
+
+dump_kube_system_pod_describes "${ARTIFACTS}/workload-cluster"
```

Comment:
can we call this from within dump_workload_cluster_logs ? Want to make it clear this is for the workload cluster only

Reply:
Makes sense. I changed it to:
```
dump_workload_cluster_logs
  dump_workload_cluster_logs_linux
  dump_workload_cluster_logs_windows
  dump_kube_system_pod_describes
```